packages/botonic-react/src/components/carousel.tsx

```diff
@@ -37,10 +49,14 @@ const StyledArrowContainer = styled.div`
   align-items: center;
   cursor: pointer;
   justify-content: ${props => props.justifyContent};
-  left: ${props => props.left}px;
-  right: ${props => props.right}px;
-  border-top-${props => props.arrow}-radius: 30px;
-  border-bottom-${props => props.arrow}-radius: 30px;
+
+  ${props => (props.left ? ` left: ${props.left}px` : '')};
+  ${props => (props.right ? ` right: ${props.right}px` : '')};
+  ${props => (props.arrow === 'right' ? 'border-top-right-radius: 30px' : '')};
+  ${props =>
+    props.arrow === 'right' ? 'border-bottom-right-radius: 30px' : ''};
+  ${props => (props.arrow === 'left' ? 'border-top-left-radius: 30px' : '')};
+  ${props => (props.arrow === 'left' ? 'border-bottom-left-radius: 30px' : '')};
```

Comment:
you could unify some conditions
``` js
${props => props.left && `left: ${props.left}px;`}
${props => props.right && `right: ${props.right}px;`}
  
${props => props.arrow === 'right' && `
  border-top-right-radius: 30px;
  border-bottom-right-radius: 30px;
`}

${props => props.arrow === 'left' && `
  border-top-left-radius: 30px;
  border-bottom-left-radius: 30px;
`}
```

Reply:
I'm going to leave it as it was because prettier leaves it with a very strange format.

![Captura de pantalla 2025-02-25 a las 9 35 53](https://github.com/user-attachments/assets/42275fb3-dc58-49a9-863e-61e77da910f5)